style.css

```diff
@@ -1901,6 +1917,10 @@ PRICE FEATURES
   font-family: "Open Sans", sans-serif;
 }
 
+.elementor-column-gap-default>.elementor-row>.elementor-column>.elementor-element-populated>.elementor-widget-wrap {
```

Comment:
This seems very generic, these chawa are all over the place..
Can't we make this more specific?